napari/layers/labels/labels.py

```diff
@@ -664,15 +665,19 @@ def _raw_to_displayed(self, raw):
         else:
             raise ValueError("Unsupported Color Mode")
 
-        if self.contour:
+        if self.contour > 0:
```

Comment:
I see a big delay when switching to 3D view and no difference in rendering. Maybe disable this in 3d mode? Until 3D section is implemented?

Reply:
done